docker-compose.prod-img.yml

```diff
@@ -0,0 +1,18 @@
+version: "3.4"
+
+services:
+  web:
+    image: ghcr.io/lucyparsons/openoversight:${DB_IMAGE_TAG}
```

Comment:
This doesn't seem to correspond to a database, should this be `DOCKER_IMAGE_TAG` instead?

Reply:
Good find, totally blanked here. Definitely should have `DOCKER` in the name, thanks!